src/test/encoding/types.h

```diff
@@ -232,9 +232,11 @@ TYPE(librbd::journal::EventEntry)
 TYPE(librbd::watch_notify::NotifyMessage)
 TYPE(librbd::watch_notify::ResponseMessage)
 
+#ifdef WITH_RBD
```

Comment:
there are other rbd types just above this (and a couple accidentally in the rgw section later too). I guess that means there are other things missing a WITH_RBD guard